3-platform/warehouse/infra/src/main/java/com/_6core/platform/warehouseinfra/adapter/driven/persistence/entity/InventoryEntity.java

```diff
@@ -0,0 +1,28 @@
+package com._6core.platform.warehouseinfra.adapter.driven.persistence.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(name = "inventories")
+public class InventoryEntity {
+  @Id private Long id;
+
+  @Column("actual_quantity")
+  private int actualQuantity;
```

Comment:
use wrapper classes pls for entity
```suggestion
  private Integer actualQuantity;
```